unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/IDCLEntity.cs

```diff
@@ -8,7 +8,8 @@ namespace DCL.Models
 {
     public interface IDCLEntity : ICleanable, ICleanableEventDispatcher
     {
-        GameObject gameObject { get; set; }
+        GameObject gameObject { get; }
+        Transform transform { get; set; }
```

Comment:
```suggestion
```
We should remove this field to keep simplicity, transform is already referenced in `gameObject`.